microsoft-edge/webview2/get-started/win32.md

```diff
@@ -589,28 +554,7 @@ Have the host app and web content communicate through `postMessage`, as follows:
 
 1. If it's not already present, paste the following code into `HelloWebView.cpp`:
 
-   ```cpp
-                  // <Step6>
-                  // Step 6 - Communication between host and web content
-                  // Set an event handler for the host to return received message back to the web content
-                  webview->add_WebMessageReceived(Callback<ICoreWebView2WebMessageReceivedEventHandler>(
-                     [](ICoreWebView2* webview, ICoreWebView2WebMessageReceivedEventArgs* args) -> HRESULT {
-                        wil::unique_cotaskmem_string message;
-                        args->TryGetWebMessageAsString(&message);
-                        // processMessage(&message);
-                        webview->PostWebMessageAsString(message.get());
-                        return S_OK;
-                     }).Get(), &token);
-                  // <Step6>
-
-                  // Schedule an async task to add initialization script that
```

Comment:
@peiche-jessica - ::: adding step 6 block will fail to get these lines below the closing Step6 tag.  Should these bottom lines remain hardcoded here?  Your change completely deleted these lines below Step6 closing tag.

Reply:
I initially placed the closing tag on the right line. This and other snippet id issues shall be fixed [on WebView2Samples.git](https://github.com/MicrosoftEdge/WebView2Samples/compare/7f06fc21e39e2aa0aec5b73e58fe6f618f2fa49e..30e6fd276923198b2c198d8252d3ae7549be74a7).